packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -504,8 +522,15 @@ export class MatchingDataAttestationGroup {
    */
   getAttestationsForBlock(fork: ForkName, notSeenAttestingIndices: Set<number>): AttestationNonParticipant[] {
     const attestations: AttestationNonParticipant[] = [];
-    const forkSeq = ForkSeq[fork];
     for (const {attestation} of this.attestations) {
+      if (
+        (isForkPostElectra(fork) && !isElectraAttestation(attestation)) ||
+        (!isForkPostElectra(fork) && isElectraAttestation(attestation))
+      ) {
+        // TODO Electra: log warning
```

Comment:
why do we wanna log a warning here? isn't it expected that the pool contains phase0 attestations shortly after fork transition? (unless we prune)

Reply:
also seems unlikely that pool contains electra attestation before the fork, do we need to check this?